### PR TITLE
Converted ClipperNacRollingShutterCamera unit test to Gtest

### PR DESCRIPTION
**Pull request by amystamile-usgs**
## Description

Updating EIS tests to Gtests. Added a Gtest in replacement of ClipperNacRollingShutterCamera's old unit test. Added ClipperNacCube fixture. 

## Motivation and Context

EIS Sprint 7/12/21

## How Has This Been Tested?

Tested with added Gtest

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/tests/ClipperNacRollingShutterCameraTests.cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  EXPECT_PRED_FORMAT2(AssertQStringsEqual, cam->instrumentNameShort(), "EIS-RSNAC");
+  EXPECT_PRED_FORMAT2(AssertQStringsEqual, cam->instrumentId(), "EIS-NAC-RS");
+
+
+  EXPECT_TRUE(cam->SetImage(145.0, 161.0));
+  EXPECT_NEAR(cam->UniversalLatitude(), 10.248688804675723, 0.0001);
+  EXPECT_NEAR(cam->UniversalLongitude(), 256.15486019464515, 0.0001);
+  EXPECT_TRUE(cam->SetUniversalGround(cam->UniversalLatitude(), cam->UniversalLongitude()));
+  EXPECT_NEAR(cam->Sample(), 290, 0.0001);
+  EXPECT_NEAR(cam->Line(), 322, 0.0001);
```
Comment (amystamile-usgs) [inline isis/tests/ClipperNacRollingShutterCameraTests.cpp]: After inputing sample and line to SetImage, the output for sample and line are doubled(for example 145->290). Not sure if this is the expected output after running SetUniversalGround.

File: isis/src/clipper/objs/ClipperNacRollingShutterCamera/ClipperNacRollingShutterCamera.truth [hunk 2]
```diff
@@ -1,31 +0,0 @@
...
-For lower left corner ...
-DeltaSample = ERROR
-DeltaLine = ERROR
-
-For lower right corner ...
-DeltaSample = ERROR
-DeltaLine = ERROR
-
-For center pixel position ...
-ERROR
```
Comment (amystamile-usgs) [inline isis/src/clipper/objs/ClipperNacRollingShutterCamera/ClipperNacRollingShutterCamera.truth:31]: Thought this was odd that the truth data was expecting all of these to output ERROR.

File: isis/tests/ClipperNacRollingShutterCameraTests.cpp (hunk 1 above)
Comment (scsides) [reply isis/tests/ClipperNacRollingShutterCameraTests.cpp]: No it should not be doing that.

File: isis/tests/Fixtures.h [hunk 3]
```diff
@@ -370,6 +370,13 @@ class CSMNetwork : public TempTestingFiles {
     void SetUp() override;
     void TearDown() override;
 };
+
+class ClipperNacCube : public DefaultCube {
```
Comment (scsides) [inline isis/tests/Fixtures.h]: Probably should add RS to the name. The FC will have its own.

**Review by scsides — DISMISSED** [review]

**Review by scsides — APPROVED** [review]
We need to remember to see how the timing table is used. It may need to be added to the NacRs fixture so the camera model can use it.

**Pull request merged**